Makefile

```diff
@@ -5,6 +5,9 @@ OS := $(shell uname -s)
 
 INTERACTIVE := $(shell [ -t 0 ] && echo 1)
 
+# BUILD_DRY_RUN determines the value of the --dry-run flag of the build command. Should be 'true' or 'false'.
+BUILD_DRY_RUN := true
```

Comment:
```suggestion
BUILD_DRY_RUN ?= true
```

Reply:
Command-line arguments (e.g. `make foo BUILD_DRY_RUN=false` always take precedence, so this is ok.